apps/condo/lang/ru/ru.json

```diff
@@ -1577,5 +1577,65 @@
   "billing.category.internet.name": "Интернет",
   "billing.category.radio.name": "Радио",
   "billing.category.intercom.name": "Домофон",
-  "billing.category.telecom.name": "Антена"
+  "billing.category.telecom.name": "Антена",
+  "banking.integration.SBBOL.name": "СББОЛ",
+  "banking.integration.1CClientBankExchange.name": "1С",
+  "banking.category.maintenance_and_current_repair_of_common_property_in_an_apartment_building.name": "Содержание и текущий ремонт общего имущества в многоквартирном доме",
+  "banking.category.apartment_building_management.name": "Управление многоквартирным домом",
+  "banking.category.overhaul.name": "Капитальный ремонт",
+  "banking.category.drainage.name": "Водоотведение",
+  "banking.category.water_supply.name": "Водоснабжение",
+  "banking.category.garbage_collection_and_disposal.name": "Вывоз и утилизация мусора",
+  "banking.category.gas.name": "Газ",
+  "banking.category.hot_water_supply.name": "Горячее водоснабжение",
+  "banking.category.additional_services.name": "Дополнительные услуги",
+  "banking.category.thermal_energy.name": "Тепловая энергия",
+  "banking.category.cold_water_supply.name": "Холодное водоснабжение",
+  "banking.category.electric_energy.name": "Электрическая энергия",
+  "banking.costItem.emergency_service.name": "Аварийное обслуживание",
+  "banking.costItem.rent.name": "Аренда",
+  "banking.costItem.banking_services.name": "Банковское обслуживание",
+  "banking.costItem.maintenance_and_operation.name": "Техническое обслуживание и эксплуатация",
+  "banking.costItem.overhaul.name": "Капитальный ремонт",
+  "banking.costItem.water_supply.name": "Водоснабжение",
+  "banking.costItem.compensation_for_damage.name": "Возмещение ущерба",
+  "banking.costItem.garbage_collection_and_disposal.name": "Вывоз и утилизация мусора",
+  "banking.costItem.gas.name": "Газ",
+  "banking.costItem.deratization_and_disinfestation.name": "Дератизация и дезинсекция",
+  "banking.costItem.materials_for_repair_and_maintenance.name": "Материалы для ремонта и обслуживания",
+  "banking.costItem.purchase_of_machinery_and_equipment.name": "Закупка техники и оборудования",
+  "banking.costItem.employee_wages.name": "Заработная плата сотрудников",
+  "banking.costItem.rental_fee.name": "Квартплата",
+  "banking.costItem.concierges.name": "Консьержи",
+  "banking.costItem.materials_for_maintenance_and_current_repair.name": "Материалы для содержания и текущего ремонта",
+  "banking.costItem.taxes.name": "Налоги",
+  "banking.costItem.equipment.name": "Оборудование",
+  "banking.costItem.maintenance_of_access_control_systems.name": "Обслуживание систем контроля управления доступом",
+  "banking.costItem.intercom_service.name": "Обслуживание домофонов",
+  "banking.costItem.maintenance_and_inspection_of_metering_devices.name": "Обслуживание и проверка приборов учета",
```

Comment:
It's not obligatory to include exact words in key, because in this way it does deviates from pattern "maintenance_of…".
Maybe inspection is included in maintenance of elevators, ventilation etc, but it does not explicitly specified in key.
Simpler to name, simpler to work with.

Reply:
too many keys similar in name, I'm not sure that I can greatly simplify them all and not lose their uniqueness and meaning

Reply:
![image](https://user-images.githubusercontent.com/1394025/221012712-ba7f0966-ed04-4749-875e-10dcfac60c3b.png)